tools/segments/src/api/query.graphql.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2025 Adobe
+ * All Rights Reserved.
+ */
+
+async function executeGraphQlQuery(query, config) {
+  const headers = {
+    'Content-Type': 'application/json',
+    'x-api-key': config['commerce.headers.cs.x-api-key'],
+    'Magento-Customer-Group': config['commerce.headers.cs.Magento-Customer-Group'],
+    'Magento-Environment-Id': config['commerce.headers.cs.Magento-Environment-Id'],
+    'Magento-Store-Code': config['commerce.headers.cs.Magento-Store-Code'],
+    'Magento-Store-View-Code': config['commerce.headers.cs.Magento-Store-View-Code'],
+    'Magento-Website-Code': config['commerce.headers.cs.Magento-Website-Code'],
+  };
+
+  const apiCall = new URL(config['commerce-core-endpoint']);
```

Comment:
I'd use `import { getConfigValue } from '../../scripts/configs.js';` and `await getConfigValue('commerce-core-endpoint')`

Reply:
Done.